pkgs/applications/networking/sync/unison/default.nix

```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation (rec {
   echo -e '\ninstall:\n\tcp $(FSMONITOR)$(EXEC_EXT) $(INSTALLDIR)' >> src/fsmonitor/linux/Makefile
   '';
 
-  makeFlags = "INSTALLDIR=$(out)/bin/" + (if enableX11 then " UISTYLE=gtk2" else "")
+  makeFlags = "INSTALLDIR=$(out)/bin/ UISTYLE=" + (if enableX11 then "gtk2" else "text")
     + (if ! ocaml.nativeCompilers then " NATIVE=false" else "");
```

Comment:
Since you're already touching this code, I suggest this cleanup (untested):
```nix
{
  makeFlags = [
    "INSTALLDIR=$(out)/bin/"
    "UISTYLE=${if enableX11 then "gtk2" else "text"}"
  ] ++ lib.optional ocaml.nativeCompilers "NATIVE=false";
}
```

Reply:
That looks good to me. I'll test it out this evening.

Reply:
Done. I tested (on macOS) and the resulting derivation is identical to the previous version (both with and without `enableX11`).